scripts/add_existing_baseyear.py

```diff
@@ -224,30 +224,19 @@ def add_power_capacities_installed_before_baseyear(n, grouping_years, costs, bas
     phased_out = df_agg[df_agg["DateOut"] < baseyear].index
     df_agg.drop(phased_out, inplace=True)
 
-    older_assets = (df_agg.DateIn < min(grouping_years)).sum()
-    if older_assets:
+    newer_assets = (df_agg.DateIn > max(grouping_years)).sum()
```

Comment:
shouldn't it be `df_agg.DateIn >= max(grouping_years)`?

Reply:
Good question... at the moment the 2020 generator is for 2016-2020. So 2020 will be included and everything >2020 will be excluded.

The opposite case of excluding 2020 might be reasonable as well, though...